extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -109,6 +111,7 @@ export abstract class AzureAuth implements vscode.Disposable {
 					Logger.error(ex.originalMessageAndException);
 				}
 			}
+			Logger.write(LogLevel.Error, ex);
```

Comment:
If it's an AzureAuthError this is duplicate the error logging - you should probably put this in an else